tests/bluetooth/controller/common/src/helper_util.c

```diff
@@ -130,13 +134,43 @@ helper_pdu_ntf_verify_func_t *const helper_pdu_ntf_verify[] = {
 	[LL_CHAN_MAP_UPDATE_IND]   = NULL,
 	[LL_LENGTH_REQ]            = NULL,
 	[LL_LENGTH_RSP]            = NULL,
+	[LL_CTE_REQ]               = NULL,
+	/* TODO (ppryga): Add verifiaction for RSP notification */
```

Comment:
nitpick spelling: verifiACtion -> verification

Reply:
Corrected, thank you for spotting.